overlay-memfault.conf

```diff
@@ -48,3 +48,6 @@ CONFIG_MEMFAULT_NCS_FW_VERSION="0.0.99-dev"
 
 # Define our own implementation of this function
 CONFIG_MEMFAULT_NCS_IMPLEMENT_METRICS_COLLECTION=n
+
+# AT command shell is useful for getting the modem out of stuck loop
+CONFIG_AT_SHELL=y
```

Comment:
Let's add this to `prj.conf` since it isn't memfault-specific

Reply:
Ah good call, done!